v2-apache/conf/extra/httpd-vhosts.conf

```diff
@@ -29,6 +29,18 @@ SSLProxyCACertificateFile ${PROXY_SSL_CA_CERT}
 
 UseCanonicalName on
 
+<VirtualHost *:${PORT}>
+    RewriteEngine On
+    RewriteCond %{REQUEST_URI} !^/\.well\-known/acme\-challenge/
+    RewriteCond %{ENV:APACHE_ALWAYS_TLS_REDIRECT} on
+    RewriteCond %{HTTP_HOST} ^(.*):${PORT}$ [NC]
```

Comment:
Why not use `SERVER_PORT` here?

Reply:
Is that a thing? Double checked docs and I can't find anything similar.

Reply:
See here https://httpd.apache.org/docs/2.4/mod/core.html#usecanonicalname and here https://httpd.apache.org/docs/current/expr.html.

Reply:
So the original problem was that HTTP_HOST comes from the client, and might include the PORT. That's why we needed two different. But forgot that we have `UseCanonicalNames` activated by default, so we can switch to SERVER_NAME and forget about HTTP_HOST.

Reply:
I see. Now I understand why you had two different rewrite rules there.